src/test/ui/const-generics/generic_arg_infer/issue-91614.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0283]: type annotations needed for `Mask<_, {_: usize}>`
+  --> $DIR/issue-91614.rs:6:13
+   |
+LL |     let y = Mask::<_, _>::splat(false);
+   |         -   ^^^^^^^^^^^^^^^^^^^ cannot infer type for type parameter `T`
+   |         |
+   |         consider giving `y` the explicit type `Mask<_, LANES>`, where the type parameter `T` is specified
```

Comment:
```suggestion
   |         consider giving `y` the explicit type `Mask<T, LANES>`, where the type parameter `T` is specified
```

Shouldn't the `_` be `T` for the suggestion to make sense (in isolation)? It says _consider giving […] the explicit type […],_ **where** _the type parameter_ **`T`** _is specified_.

Reply:
Yea this error message is sort of weird (especially how it is `_` for the type and `LANES` for the const despite both being inference var)